rush.json

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
-  "rushVersion": "5.120.2",
-  "pnpmVersion": "7.32.2",
-  "nodeSupportedVersionRange": "^18.12.0 || ^20.9.0",
+  "rushVersion": "5.139.0",
+  "pnpmVersion": "9.12.2",
```

Comment:
bumping pnpm version is big, do you want to do it in this pr?

Reply:
Reverted to pnpm 7